Encounter/encounter_type.md

```diff
@@ -0,0 +1,97 @@
+# Encounter Type
+
+## Overview
+
+* A visit can be associated with one or more encounters. OpenMRS allows you to define multiple encounter types.
+
+* You could define encounter type for locations such as Pharmacy, Lab, Consultation room or for actions such as admission or discharge.
+
+## Available operations. 
+
+1. [List encounter types](#list-encounter-types)
+2. [Create an encounter type](#create-a-encounter-type)
+3. [Update an encounter type](#update-a-encounters-type)
+4. [Delete an encounter type](#delete-a-encounters-type)
+
+
+### List encounter types
+
+* #### List all non-voided encounter types.
```

Comment:
Encounter types are metadata, so they are [retired](https://docs.openmrs.org/datamodel/openmrs-refapp-data-model-1.9.html#encounter_type.retired), not voided.